keadm/cmd/keadm/app/cmd/util/common.go

```diff
@@ -165,15 +166,22 @@ func RunningModule() (types.ModuleRunning, error) {
 
 // GetLatestVersion return the latest non-prerelease, non-draft version of kubeedge in releases
 func GetLatestVersion() (string, error) {
-	//Download the tar from repo
-	versionURL := "curl -k " + latestReleaseVersionURL
-	cmd := exec.Command("sh", "-c", versionURL)
-	latestReleaseData, err := cmd.Output()
+	// curl https://kubeedge.io/latestversion
+	resp, err := http.Get(latestReleaseVersionURL)
```

Comment:
By my test, throuth method `http.Get(latestReleaseVersionURL)` result is equal to directly exec shell command `"curl -k " + latestReleaseVersionURL`
Is this refactor work OK? @fisherxu 
If OK, but I  also think we need to open another PR to refactor this function, but not mix it with other functions.

Reply:
Sure, I can make an issue for this and put it in another PR

Reply:
I have removed this code from this PR and made #3385 to address this - I can work on it if @fisherxu agrees.